Splunk Phantom/README.md

```diff
@@ -1,25 +1,33 @@
+
+
+
 ## DomainTools Phantom Playbooks
 
-Playbooks and automation scripts for Splunk Phantom.
+Working playbooks and automation scripts for Splunk Phantom. 
 
-### Current Playbooks In This Repo
+#### Installation
+For the DomainTools playbooks in this repo, an asset called `domaintoolsiriscreds` is needed with your DomainTools username and key. The playbooks expect this asset to exist to make the API calls needed for the playbook data. Installation instructions for each playbook in the README at the root of the playbook’s directory.
```

Comment:
Should "instructions for each playbook in .." be "instructions for each playbook _is_ in .."

Reply:
Yes